### PR TITLE
[SCB-1407] delete sensitive log

### PR DESCRIPTION
**Pull request by GuoYL123**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java [hunk 1]
```diff
@@ -116,9 +116,6 @@ public static ConcurrentCompositeConfiguration createLocalConfig() {
     }
 
     LOGGER.info("create local config:");
-    for (ConfigModel configModel : loader.getConfigModels()) {
-      LOGGER.info(" {}.", configModel.getUrl());
-    }
```
Comment (yhs0092) [inline foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java]: Maybe it's not a good idea to remove this log.
This log is often checked when config related problems occur.

Comment (GuoYL123) [reply foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java]: Users decide whether to print through configuration

File: foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java [hunk 2]
```diff
@@ -58,6 +59,8 @@
 
   private static final String MICROSERVICE_CONFIG_LOADER_KEY = "cse-microservice-config-loader";
 
+  private static final String IS_PRINT_URL = "cse.config.isPrintUrl";
```
Comment (liubao68) [inline foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java]: Use "servicecomb.config.log.verbose" is better, and can control other logs in future.

File: foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java [hunk 3]
```diff
@@ -117,7 +120,11 @@ public static ConcurrentCompositeConfiguration createLocalConfig() {
 
     LOGGER.info("create local config:");
     for (ConfigModel configModel : loader.getConfigModels()) {
-      LOGGER.info(" {}.", configModel.getUrl());
+      DynamicBooleanProperty isPrintUrl = DynamicPropertyFactory.getInstance()
```
Comment (liubao68) [inline foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java]: Do not write this in loop. Maybe append all info to a StringBuilder and log it in one line.

File: foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java (hunk 2 above)
Comment (GuoYL123) [reply foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java]: done

File: foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java (hunk 3 above)
Comment (GuoYL123) [reply foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java]: add toString

File: foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java [hunk 4]
```diff
@@ -116,8 +119,10 @@ public static ConcurrentCompositeConfiguration createLocalConfig() {
     }
 
     LOGGER.info("create local config:");
-    for (ConfigModel configModel : loader.getConfigModels()) {
-      LOGGER.info(" {}.", configModel.getUrl());
+    boolean isPrintUrl = DynamicPropertyFactory.getInstance()
+        .getBooleanProperty(IS_PRINT_URL, true).get();
+    if (isPrintUrl) {
+      LOGGER.info(" {}.", loader.getConfigModels());
```
Comment (liubao68) [inline foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java]: load.getConfigModels is a List.  It's a bad practise to invoke toString of a List. Maybe
```
StringUtils.join(loader.getConfigModels(), ",") 
```
is better.

Comment (GuoYL123) [reply foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java]: done.

**Pull request merged**